### PR TITLE
docs(sites): add aquagruppen.se

### PR DESCRIPTION
**Pull request by elitan**
added aquagruppen.se to docs/sites.yml

### CODE REVIEW

File: docs/sites.yml [hunk 1]
```diff
@@ -4516,3 +4516,14 @@
   built_by: Nick Taylor
   built_by_url: https://www.iamdeveloper.com
   featured: false
+- title: AquaGruppen Vattenfilter
+  url: https://aquagruppen.se
+  main_url: https://aquagruppen.se/
+  description: >
+    Vattenfilter och vattenrening (Water filter and water treatment products in Sweden)
```
Comment (DSchau) [inline docs/sites.yml]:
```suggestion
    Water filter and water treatment products in Sweden
```

Let's just default to English here, if we can!

**Comment by elitan** [comment]
It is ok

**Comment by elitan** [comment]
@sidharthachatterjee Thanks for merging! Funny how things works out 😃

**Comment by sidharthachatterjee** [comment]
@elitan Johan! Great to see you here ☺️

**Pull request merged**